libs/docs/cdk/focusable-grid/examples/default-example/default-example.component.html

```diff
@@ -1,5 +1,10 @@
-<div fdkFocusableGrid wrapHorizontally (itemFocused)="itemFocused($event)">
-    <div *ngFor="let list of [].constructor(3); let rowIndex = index" fdkFocusableList navigationDirection="grid">
+<div fdkFocusableGrid wrapHorizontally (itemFocused)="itemFocused($event)" (rowFocused)="rowFocused($event)">
```

Comment:
Now when I focus a cell in the first row, then move focus to a cell in the next row, the whole row is focused, and I cannot see a way to go back to just focusing an individual cell. Is this intentional?

Reply:
Yea, it's intended and to focus cell inside a focusable row use F2 key, I mentioned it in the docs.